src/components/OpenPopup.jsx

```diff
@@ -24,7 +24,7 @@ class OpenPopup extends Component {
       <div className="App">
         <button onClick={this.togglePopup}>show popup</button>
         {this.state.showPopup ? (
-          <Popup score="89" closePopup={this.togglePopup} />
+          <Popup score="89"  nameUser={localStorage.getItem("username")}  closePopup={this.togglePopup} />
```

Comment:
Please add username in the state at state init / componentDidMount

Reply:
also change variable from nameUser to username